.gitignore

```diff
@@ -1,5 +1,28 @@
-_site
-.sass-cache
-.jekyll-cache
+# Jekyll
+_site/
+*-cache/
 .jekyll-metadata
-vendor
+
+# Ruby
+.bundle/
+.byebug_history
+.ruby-gemset
+.ruby-version
+*.gem
+Gemfile.lock
```

Comment:
@crduffy Can this line be removed so that we allow this lock file to be committed. That way we know the dependencies (jekyll) won't break on a minor version for new contributors.